src/components/layout/tables/AntTable/AntTable.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+
+import {
+  BookmarkIcon,
+  BorderOuterIcon,
+  FileCodeIcon,
+  RefreshAlertIcon,
+} from '../../../icons';
+import AntRow from '../AntRow/AntRow';
+import './styles.css';
+
+function AntTable({ antIds, reload }: { antIds: string[]; reload: boolean }) {
+  // eslint-disable-next-line
+  const [tableItems, setTableItems] = useState([<></>]);
+
+  useEffect(() => {
+    updateTableItems()
+      .then((items) => setTableItems(items))
+      .catch((err: Error) => console.error(err));
+  }, [antIds, reload]);
+
+  async function updateTableItems() {
+    const items = [];
+    for (const id of antIds) {
+      items.push(
+        <AntRow
+          antId={id}
+          bgColor={'#1E1E1E'}
+          textColor={'var(--text-white)'}
+        />,
+      );
+    }
+    return items;
+  }
```

Comment:
if you're going to go this route (without async calls in the table and only at the row level) - then you can remove this entirely and just do a JSX block below